imports/ui/report-explorer/report-explorer.jsx

```diff
@@ -87,13 +99,19 @@ class ReportExplorer extends Component {
                   />)
                 }
                 name={'report'}
-              /> : (<NoItemMsg item={'report'} />)
+              /> : (<NoItemMsg item={'report'} ButtonOption />)
```

Comment:
This doesn't work because you didn't rename it when you renamed the component's prop to `buttonOption`